src/app/backend/dashboard.go

```diff
@@ -32,6 +32,7 @@ var (
 )
 
 func main() {
+	//TODO(maciaszczykm): Resolve issue with logs (ERROR: logging before flag.Parse).
```

Comment:
I think to solve this problem we can ditch glog package and start using just normal golang log package. What do you think?